cmd/kubeless/function_test.go

```diff
@@ -143,7 +147,7 @@ func TestGetFunctionDescription(t *testing.T) {
 	}
 
 	// Given parameters should take precedence from default values
-	result3, err := getFunctionDescription("test", "default", "file.handler2", "function2", "dependencies2", "runtime2", "test_topic", "", "test-image2", "256Mi", false, []string{"TEST=2"}, []string{"test=2"}, expectedFunction)
+	result3, err := getFunctionDescription("test", "default", "file.handler2", "function2", "dependencies2", "runtime2", "test_topic", "", "test-image2", "256Mi", false, false, 8080, []string{"TEST=2"}, []string{"test=2"}, expectedFunction)
```

Comment:
change the values so we check that the new parameters are persisted/overwritten by the default/new function.